python/glide-shared/glide_shared/config.py

```diff
@@ -500,23 +530,21 @@ def __init__(
             read_from=read_from,
             request_timeout=request_timeout,
             reconnect_strategy=reconnect_strategy,
+            database_id=database_id,
```

Comment:
Does this value need to be validated? It is validated to be >= 0 in other languages.